airframe-sql/src/test/resources/wvlet/airframe/sql/standard/queries.yml

```diff
@@ -192,3 +192,5 @@
       select 1 as c1
 - sql: |
       select count(1) from t having count(distinct id) > 5
+- sql: | 
+      select EXTRACT(QUARTER FROM FROM_UNIXTIME(time)), COUNT(*) FROM a GROUP BY EXTRACT(QUARTER FROM FROM_UNIXTIME(time))
```

Comment:
Good. Could you add more example queries to have the full coverage? 

Btw, I found GitHub copilot is really good at generating such test queries by looking at the code diff.

Reply:
I added additional tests to cover all intervals. I also pushed a fix that prevented a test from succeeding.